.github/scripts/nft_boost/actions.utils.ts

```diff
@@ -1,6 +1,7 @@
 import axios from 'axios'
 import { Address, Block, getAddress } from 'viem'
-import { BackersManagerAbi, BuilderRegistryAbi, GaugeAbi } from '../../../src/lib/abis/v2'
+import { getAbi } from '../../../src/lib/abis/v2'
```

Comment:
```suggestion
import { getAbi } from '@/lib/abis/v2'
```

Reply:
this is not going to work because it's not inside the project